wd_helper.au3

```diff
@@ -990,8 +955,7 @@ Func _WD_ElementOptionSelect($sSession, $sStrategy, $sSelector, $sStartElement =
 		_WD_ElementAction($sSession, $sElement, 'click')
 	EndIf
 
-	Return SetError(@error, @extended)
-
+	Return SetError(__WD_Error($sFuncName, @error), $_WD_HTTPRESULT)
```

Comment:
Need to review for all instances where setting @extended to $_WD_HTTPRESULT

Reply:
Done in wd_helper.au3